gflib/text.h

```diff
@@ -269,6 +269,13 @@
 
 #define TEXT_SPEED_FF 0xFF
 
+enum ColorMode 
+{
+    FgColor,
+    ShadowColor,
+    BgColor
```

Comment:
Use the naming convention of `CONSTANT_NAME`. The enum name also isn't necessary, it should be obvious what they're for

Reply:
Done